scripts/update_manifest.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python
+
+import json
+import requests
+
+with open("src/main/resources/manifest.json", "r") as manifestFile:
+    manifestStr = manifestFile.read()
+    manifest = json.loads(manifestStr)
+    for entry in manifest:
+        packageName = entry["packageName"]
+        currentVersion = entry["version"]
+        packageInfo = requests.get(f"https://packages.simplifier.net/{packageName}").json()
+        latestVersion = packageInfo["dist-tags"]["latest"]
+        print(f"{packageName}: {currentVersion} -> {latestVersion}")
```

Comment:
we have this for logging?

Reply:
Yeah just thought it would be useful to see how the package versions have changed